src/web/src/components/Posts/Post.tsx

```diff
@@ -61,6 +62,12 @@ const useStyles = makeStyles((theme: Theme) =>
         padding: '1em 0 1em 0',
       },
     },
+    video: {
+      display: 'block',
+      margin: 'auto',
+      width: '85%',
```

Comment:
Why 85% and not fill the available space in the parent container?

Reply:
I thought that the 85% looked better that covering the whole container. It seemed that the video was too close to the summary information on the right.